artemis-server/src/main/java/org/apache/activemq/artemis/core/server/ActiveMQServerLogger.java

```diff
@@ -1621,4 +1621,7 @@ void noQueueIdDefined(org.apache.activemq.artemis.api.core.Message message,
 
    @LogMessage(id = 224139, value = "Failed to stop bridge: {}", level = LogMessage.Level.ERROR)
    void errorStoppingBridge(String bridgeName, Exception e);
+
+   @LogMessage(id = 224140, value = "Clearing bindings on cluster-connection {} failed to remove binding {}: {}", level = LogMessage.Level.WARN)
+   void clusterConnectionFailedToRemovingBindingOnClear(String clusterConnection, String binding, String exceptionMessage);
```

Comment:
I'd feel clusterConnectionFailedToRemoveBindingOnClear would be more grammatically correct

Reply:
Fixed.